controllers/tests/array_action/svc/array_mediator_svc_test.py

```diff
@@ -99,6 +105,200 @@ def test_close(self):
         self.svc.disconnect()
         self.svc.client.close.assert_called_with()
 
+    def test_get_replication_success(self):
+        replication_request = ReplicationRequest(OBJECT_INTERNAL_ID, OTHER_OBJECT_INTERNAL_ID, SYSTEM_ID, COPY_TYPE,
+                                                 REPLICATION_TYPE_MIRROR)
+        rcrelationships_to_return = \
+            [Munch({svc_settings.RCRELATIONSHIP_STATE_ATTR_NAME: RCRELATIONSHIP_STATE_READY,
+                    svc_settings.RCRELATIONSHIP_COPY_TYPE_ATTR_NAME: svc_settings.RCRELATIONSHIP_COPY_TYPE,
+                    svc_settings.RCRELATIONSHIP_MASTER_CLUSTER_ID_ATTR_NAME: "",
+                    svc_settings.RCRELATIONSHIP_PRIMARY_ATTR_NAME: False,
+                    svc_settings.RCRELATIONSHIP_ID_ATTR_NAME: svc_settings.DUMMY_INTERNAL_ID1,
+                    svc_settings.RCRELATIONSHIP_NAME_ATTR_NAME: ""})]
+        self.svc.client.svcinfo.lsrcrelationship = Mock()
```

Comment:
can we `Mock()` everything once in some preparation method? (BTW, since `self.svc.client` is mocked I don't think we really need those Mocks)

Reply:
Removed mocks for svcinfo and svctask.